cwms_radar_api/build.gradle

```diff
@@ -12,8 +12,8 @@ dependencies {
     implementation "javax.xml.bind:jaxb-api:2.3.1"
     implementation "com.sun.xml.bind:jaxb-impl:3.0.2"
     implementation "com.sun.xml.bind:jaxb-core:3.0.2"
-    implementation "mil.army.usace.hec:cwms-db-dao:9.0.2"
-    implementation( "mil.army.usace.hec:cwms-db-jooq:9.0.2"){
+    implementation "mil.army.usace.hec:cwms-db-dao:9.0.4"
```

Comment:
Has it dawned on any of us much much of an almost circular dependency this is?

Not saying remove it it. Just felt like it needed to be said.